src/components/SystemPreferences.vue

```diff
@@ -132,4 +178,8 @@ div.slider-label {
   margin-top: 6px;
 }
 
+.vue-slider >>> .vue-slider-process {
```

Comment:
Is `>>> X` the same as `> :not(X)` ? Because it works live, but it doesn't work in a fiddle env like https://nightcatsama.github.io/vue-slider-component/#/basics/process

What CSS magic is making this work?

Reply:
That's Vue magic, see [doc](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors).  Basically it's useful for digging into components.